src/NServiceBus.Core.Tests/Unicast/RunningEndpointInstanceTest.cs

```diff
@@ -21,8 +22,32 @@ public async Task ShouldAllowMultipleStops()
                 null);
 
             await testee.Stop();
+            
 
             Assert.That(async () => await testee.Stop(), Throws.Nothing);
+        }       
+        
+        [Test]
+        public async Task ThrowsExceptionWhenNotRunning()
```

Comment:
The name doesn't include what's triggering the exception so perhaps rename to "ShouldThrowIfUsedAfterBeingStopped" ?